lib/deps/ajax/ajaxCore.js

```diff
@@ -87,19 +86,7 @@ function ajax(options, callback) {
 
   return request(options, function (err, response, body) {
     if (err) {
-      // Node request only ever fires error or response, never both
```

Comment:
Nah the comment was there only to mention that the following code path would not be followed in node